### PR TITLE
Enable keyboard support

### PR DESCRIPTION
**Pull request by luckysoni**
https://app.asana.com/0/1201325186562318/1203201686089275/f
[SPIKE] Ss on SR/Keyboard can move coordinates on graph

It turns out that any elements nested inside the `<canvas>` are not visible to the user but are accessible to a screen-reader and focusable (if interactive) by the keyboard. This knowledge was key in making these graphs support keyboard interaction.
I created a one-to-one mapping between nested `button`s and points on the graph. Once a `button` is focused, the mapped point on the graph renders its focus state and listens for keyboard events.

Peek 2023-01-23 22-10

### CODE REVIEW

File: src/helpers/paper-util.js [hunk 1]
```diff
@@ -352,6 +439,45 @@ const PaperUtil = {
       func(inGridBoundsPoint, ...args)
     }
 
+    this.getTargetPointForKeyboardEvent = (event) => {
+      const focusedPath = this.getFocusedStartingPoint()
+      const ALLOWED_KEYS = ['up', 'down', 'left', 'right']
+      if(focusedPath !== null && focusedPath !== undefined && ALLOWED_KEYS.includes(event.key)) {
+        const gridPoint = this.fromViewCoordinateToGrid(focusedPath.position)
+        const targetPoint = (() => {
+          switch(event.key) {
```
Comment (eborden) [inline src/helpers/paper-util.js:448]: Are these `event.key` values stable? Are there any compatibility modes to consider?

Comment (luckysoni) [reply src/helpers/paper-util.js:448]: Yes, these are stable as per [paperjs](http://paperjs.org/reference/keyevent/#key)

File: src/helpers/paper-util.js [hunk 2]
```diff
@@ -352,6 +439,45 @@ const PaperUtil = {
...
+            case 'up':
+              return {
+                ...gridPoint,
+                x: gridPoint.x,
+                y:  gridPoint.y + graphSettings.stepY
```
Comment (eborden) [inline src/helpers/paper-util.js:453]: Do these operations need a "clamp" so that they can't fall off the grid?

Comment (luckysoni) [reply src/helpers/paper-util.js:453]: The function runs the point through `getClosestInGridPoint` before returning the result. `getClosestInGridPoint` ensures that these are in bound.

**Review by eborden — APPROVED** [review]

**Pull request merged**